doc/source/_static/custom.css

```diff
@@ -61,3 +61,39 @@ img.logo {
 .prev-next-top {
   margin-bottom: 1em;
 }
+
+/* Sidebar TOC and headers */
+
+div.sphinxsidebarwrapper div {
+    margin-bottom: .8em;
+}
+div.sphinxsidebar h3 {
+    font-size: 1.3em;
+    padding-top: 0px;
+    font-weight: 800;
+    margin-left: 0px !important;
+}
+
+div.sphinxsidebar p.caption {
+    font-size: 1.2em;
+    margin-bottom: 0px;
+    margin-left: 0px !important;
+    font-weight: 800;
+    color: gray;

```

Comment:
We need to increase the contrast to at least #767676 color for accessibility.

FYI. I use AXE extension on Chrome to test. http://accessible-colors.com/ is useful for if I have... what do I need...

Reply:
whoah! that's cool! I'd never heard about that, OK will change it in a sec